spec/raven/logger_spec.cr

```diff
@@ -1,12 +1,16 @@
 require "../spec_helper"
+require "log/spec"
 
 describe Raven::Logger do
   it "should log to a given IO" do
-    io = IO::Memory.new
+    log = Raven::Logger.for(Raven::Logger::PROGNAME)
 
-    logger = Raven::Logger.new(io)
-    logger.fatal("Oh noes!")
+    Raven::Logger.capture(builder: Raven::Logger.builder) do |logs|
+      log.info { "Oh YAZ!" }
+      log.fatal { "Oh noes!" }
 
-    io.to_s.should match(/FATAL -- sentry: Oh noes!\n\Z/)
```

Comment:
Checking `IO` for written messages is essential to this spec.

Reply:
Sorry, my head is in many places today. :blush:

Reply:
No worries but it still doesn't do what's suppose to ;)

Reply:
Right, so it should test the exact output in the logs? E.g. `FATAL -- sentry: Oh noes!`? Because right now it does test what's described in spec: "should log to a given IO".

Reply:
Well, not rly since it tests whether the logs are sent to the log _backend_, not the _io_ itself. I guess it's fine like this though, I'd only ask you to change the test name to reflect that.

Reply:
Ok, I'll add another spec with and IO backend as well.